python/hail/docs/tutorials/hail-overview.ipynb

```diff
@@ -1118,9 +1118,9 @@
    "source": [
     "## Rare variant analysis\n",
     "\n",
-    "Hail doesn't yet have rare variant kernel-based methods, but we have [linear](https://hail.is/hail/hail.VariantDataset.html#hail.VariantDataset.linreg_burden) and [logistic](https://hail.is/hail/hail.VariantDataset.html#hail.VariantDataset.logreg_burden) burden tests. \n",
+    "Here we'll demonstrate how one can use the expression language to group and count by any arbitrary properties in variant or sample annotations. Hail also implements the sequence kernel association test (SKAT).\n",
     "\n",
-    "We won't be showing those here, though. Instead, we'll demonstrate how one can use the expression language to group and count by any arbitrary properties in variant or sample annotations."
+    "Instead, we'll demonstrate how one can use the expression language to group and count by any arbitrary properties in variant or sample annotations."
```

Comment:
Sorry, I should have been more specific: my suggestion was meant to replace both 1121 and 1123. I'll approve, but change the end 1121 from `(SKAT).\n",` to `(SKAT)."` and delete 1122 and 1123 before merging.